_templates/base.html

```diff
@@ -0,0 +1,5 @@
+{% extends "!base.html" %}
+{% block scripts %}
+{{ super() }}
+<feedback-button project="hiz6sjmrnx">Send feedback</feedback-button>
```

Comment:
```suggestion
<feedback-button project="hiz6sjmrnx" button-position="bottom-right" modal-position="bottom-right" button-style="dark">Send feedback</feedback-button>
```

To make the button float and a bit more vissible.